### ISSUE TITLE
Snapshots/20210414

### ISSUE THREAD

**Issue by dvdhrm** [osbuild/rpmrepo#2]
This is a tracking issue for rpmrepo snapshots tagged 20210414.

**Comment by dvdhrm** [comment]
`Job "daherrma-snapshot-20210414" with id "df086a61-5816-46ce-9f97-5731609c0c7a" will be submitted`

Snapshot jobs are running on AWS Batch. The target suffix is `-20210414` (i.e., today).

**Comment by dvdhrm** [comment]
64 jobs succeeded, one job failed:

```
++ cat ./repo/f34-x86_64-devel.json
--
+ TARGET_JSON='{
"base-url": "https://dl01.fedoraproject.org/pub/fedora/linux/development/34/Everything/x86_64/os/",
"platform-id": "f34",
"snapshot-id": "f34-x86_64-devel",
"storage": "public"
}'

[...]

(22307/63583): libi2c-devel-4.2-3.fc34.i686.rpm 162 kB/s \|  11 kB     00:00
(22308/63583): libi2c-devel-4.2-3.fc34.x86_64.r 149 kB/s \|  11 kB     00:00
(22309/63583): libib-util-3.0.7.33374-3.fc34.x8 167 kB/s \|  12 kB     00:00
(22310/63583): libibcommon-1.2.0-22.fc34.i686.r 415 kB/s \|  28 kB     00:00
(22311/63583): libibcommon-1.2.0-22.fc34.x86_64 413 kB/s \|  28 kB     00:00
[MIRROR] libguestfs-man-pages-uk-1.45.2-2.fc34.noarch.rpm: Curl error (28): Timeout was reached for https://dl01.fedoraproject.org/pub/fedora/linux/development/34/Everything/x86_64/os/Packages/l/libguestfs-man-pages-uk-1.45.2-2.fc34.noarch.rpm [Resolving timed out after 30001 milliseconds]
[MIRROR] libhubbub-devel-0.3.7-3.fc34.x86_64.rpm: Curl error (28): Timeout was reached for https://dl01.fedoraproject.org/pub/fedora/linux/development/34/Everything/x86_64/os/Packages/l/libhubbub-devel-0.3.7-3.fc34.x86_64.rpm [Resolving timed out after 33215 milliseconds]
[MIRROR] libibcommon-devel-1.2.0-22.fc34.i686.rpm: Curl error (28): Timeout was reached for https://dl01.fedoraproject.org/pub/fedora/linux/development/34/Everything/x86_64/os/Packages/l/libibcommon-devel-1.2.0-22.fc34.i686.rpm [Resolving timed out after 32577 milliseconds]
[MIRROR] libguestfs-man-pages-uk-1.45.2-2.fc34.noarch.rpm: Curl error (6): Couldn't resolve host name for https://dl01.fedoraproject.org/pub/fedora/linux/development/34/Everything/x86_64/os/Packages/l/libguestfs-man-pages-uk-1.45.2-2.fc34.noarch.rpm [Could not resolve host: dl01.fedoraproject.org]
[FAILED] libguestfs-man-pages-uk-1.45.2-2.fc34.noarch.rpm: No more mirrors to try - All mirrors were already tried without success
Traceback (most recent call last):
File "/usr/lib64/python3.9/runpy.py", line 197, in _run_module_as_main
return _run_code(code, main_globals, None,
File "/usr/lib64/python3.9/runpy.py", line 87, in _run_code
exec(code, run_globals)
File "/osb/workdir/rpmrepo/src/ctl/__main__.py", line 13, in <module>
sys.exit(global_main.run())
File "/osb/workdir/rpmrepo/src/ctl/cli.py", line 216, in run
ret = CliPull(self).run()
File "/osb/workdir/rpmrepo/src/ctl/cli.py", line 47, in run
cmd.pull()
File "/osb/workdir/rpmrepo/src/ctl/pull.py", line 97, in pull
raise RuntimeError(f"Failed during dnf reposync with exitcode '{ret}'")
RuntimeError: Failed during dnf reposync with exitcode '1'
```

**Comment by ondrejbudai** [comment]
@dvdhrm what's the plan with the failure? did you reschedule the job?

**Comment by dvdhrm** [comment]
> @dvdhrm what's the plan with the failure? did you reschedule the job?

See my previous comment at the bottom ;) I already submitted a fixup job.

**Comment by dvdhrm** [comment]
All 63 snapshots are complete. 2 singletons were not captured (they are static repositories).

```
$ curl https://rpmrepo.osbuild.org/v2/enumerate | jq '[.[] | select(test("20210414"))]'
[
  "cs8-aarch64-appstream-20210414",
  "cs8-aarch64-baseos-20210414",
  "cs8-ppc64le-appstream-20210414",
  "cs8-ppc64le-baseos-20210414",
  "cs8-x86_64-appstream-20210414",
  "cs8-x86_64-baseos-20210414",
  "el8-aarch64-appstream-n8.4-20210414",
  "el8-aarch64-appstream-n8.5-20210414",
  "el8-aarch64-appstream-r8.2-20210414",
  "el8-aarch64-appstream-r8.3-20210414",
  "el8-aarch64-baseos-n8.4-20210414",
  "el8-aarch64-baseos-n8.5-20210414",
  "el8-aarch64-baseos-r8.2-20210414",
  "el8-aarch64-baseos-r8.3-20210414",
  "el8-ppc64le-appstream-n8.4-20210414",
  "el8-ppc64le-appstream-n8.5-20210414",
  "el8-ppc64le-appstream-r8.2-20210414",
  "el8-ppc64le-appstream-r8.3-20210414",
  "el8-ppc64le-baseos-n8.4-20210414",
  "el8-ppc64le-baseos-n8.5-20210414",
  "el8-ppc64le-baseos-r8.2-20210414",
  "el8-ppc64le-baseos-r8.3-20210414",
  "el8-s390x-appstream-n8.4-20210414",
  "el8-s390x-appstream-n8.5-20210414",
  "el8-s390x-appstream-r8.2-20210414",
  "el8-s390x-appstream-r8.3-20210414",
  "el8-s390x-baseos-n8.4-20210414",
  "el8-s390x-baseos-n8.5-20210414",
  "el8-s390x-baseos-r8.2-20210414",
  "el8-s390x-baseos-r8.3-20210414",
  "el8-x86_64-appstream-n8.4-20210414",
  "el8-x86_64-appstream-n8.5-20210414",
  "el8-x86_64-appstream-r8.2-20210414",
  "el8-x86_64-appstream-r8.3-20210414",
  "el8-x86_64-baseos-n8.4-20210414",
  "el8-x86_64-baseos-n8.5-20210414",
  "el8-x86_64-baseos-r8.2-20210414",
  "el8-x86_64-baseos-r8.3-20210414",
  "el8-x86_64-rt-n8.4-20210414",
  "el8-x86_64-rt-n8.5-20210414",
  "el8-x86_64-rt-r8.2-20210414",
  "el8-x86_64-rt-r8.3-20210414",
  "el9-aarch64-appstream-n9.0-20210414",
  "el9-aarch64-baseos-n9.0-20210414",
  "el9-ppc64le-appstream-n9.0-20210414",
  "el9-ppc64le-baseos-n9.0-20210414",
  "el9-s390x-appstream-n9.0-20210414",
  "el9-s390x-baseos-n9.0-20210414",
  "el9-x86_64-appstream-n9.0-20210414",
  "el9-x86_64-baseos-n9.0-20210414",
  "el9-x86_64-rt-n9.0-20210414",
  "f33-aarch64-fedora-20210414",
  "f33-aarch64-fedora-modular-20210414",
  "f33-aarch64-updates-released-20210414",
  "f33-aarch64-updates-released-modular-20210414",
  "f33-x86_64-updates-released-20210414",
  "f33-x86_64-updates-released-modular-20210414",
  "f34-aarch64-devel-20210414",
  "f34-aarch64-devel-modular-20210414",
  "f34-x86_64-devel-20210414",
  "f34-x86_64-devel-modular-20210414",
  "f35-x86_64-rawhide-20210414",
  "f35-x86_64-rawhide-modular-20210414"
]
```
